src/jupyterhub_config/custom_spawner.py

```diff
@@ -118,8 +120,8 @@ def _ensure_user_directory(self, user_dir: Path, username: str):
             # Change the directory's ownership to the user
             os.chown(user_dir, uid, gid)
 
-            # Set directory permissions to 700: Owner (rwx), Group (---), Others (---)
-            os.chmod(user_dir, 0o700)
+            # Set directory permissions to 750: Owner (rwx), Group (r-x), Others (---)
+            os.chmod(user_dir, 0o750)
```

Comment:
requested by Chirs that by default the team should be able to see each other's notebook. It should be fine since the system will be mainly used by internal users in the near future.